src/orchestrator/streaming.rs

```diff
@@ -277,6 +278,7 @@ async fn streaming_output_detection_task(
         // Get the default threshold to use if threshold is not provided by the user
         let default_threshold = detector_config.default_threshold;
         let threshold = detector_params.threshold().unwrap_or(default_threshold);
+        detector_params.remove("threshold");
```

Comment:
Since you've setup a constant for this parameter, should we use the constant here instead*?